README.md

```diff
@@ -2,21 +2,26 @@
 
 Run Jest against Lightning web components in a Salesforce DX workspace environment.
 
-## Master vs. Prerelease
+## Versions
 
-To test your components against Salesforce production instances, use the master branch. The latest release off of this branch is tagged with the npm dist-tag of `latest`.
+To test against the latest Salesforce production instances, use the npm tag appropriate for the current release, e.g.:
 
-To test against sandbox instances, use the `prerelease` branch and the version tagged as `prerelease`.
+```
+yarn add -D @salesforce/sfdx-lwc-jest@winter22
+yarn add -D @salesforce/sfdx-lwc-jest@spring22
```

Comment:
I don't see those releases being published to NPM: https://www.npmjs.com/package/@salesforce/sfdx-lwc-jest.
It's certainly something we should automate.

Reply:
I published npm dist-tags for winter22 and spring22 (which incidentally, are the same version).

In terms of automating, it would be nice to have unified scripts so we don't have to duplicate for each of our projects.